test/src/sdk-harness/test_artifacts/methods_contract/src/main.sw

```diff
@@ -6,7 +6,7 @@ use std::result::*;
 use std::identity::*;
 use std::chain::auth::*;
 use std::option::*;
-use std::assert::*;
+use std::revert::*;
```

Comment:
Maybe see how many of these `*` imports can be cleaned up. Some of them were required at one point and no longer are.

Reply:
`std::result::*` wasn't being used, removed.